pcsx2/Recording/InputRecordingFile.h

```diff
@@ -68,6 +70,8 @@ class InputRecordingFile
 	bool FromSaveState();
 	// Increment the number of undo actions and commit it to the recording file
 	void IncrementUndoCount();
+	bool IsFileOpen();
+	void SetUndoCount(long undoCount);
```

Comment:
Manually changing the undo count of a recording is something that should not be supported by the program.

Reply:
Elaboration: having this could lead to people falsely presenting their recordings when attempting to post it to sites. We want to maintain that recordings made by the program are authentic and that the values are valid representation of the actual effort put into it.

Reply:
Yeah that's fair, I went back and forth on it a bit.  I'll remove it, but at the end of the day, if someone wants to lie and modify their undocount, they are going to find a way, and I don't think we need to be the arbiters TAS accuracy :P

Reply:
Yes, but I'd rather the fault of said action be 100% on the user and not have it whatsoever attributed to the program allowing them to easily do so.